### ISSUE TITLE
(group-exports) Option to never put the export next to the declaration

### ISSUE THREAD

**Issue by shoelessness** [benmosher/eslint-plugin-import#1871]
I'd like to see an option for always having named exports be specified separately to their definition. e.g.

**Fail**
```js
export const foo = 'bar';
```

**Pass**

```js
const foo = 'bar';

export { foo };
```

I'm basically trying to mitigate the workflow of moving the export keyword to the bottom of the file if another named export is added.

**Comment by ljharb** [comment]
I'm confused, why would
```js
export const foo = 'bar';
export const baz = 'quux';
```
not work in that case?

**Comment by shoelessness** [comment]
It's not that it wouldn't work, it's a code style preference to group the exports together always at the bottom of a file.

For instance, given your example, I would want to see:

```js
const foo = 'bar';
const baz = 'quux';

export { foo, baz };
```

This is already supported by simply having the `group-exports` rule turned on, but I'm looking to avoid the refactor pattern of adding another export to a file.

So if you start the file with:

```js
export const foo = 'bar';
```

and then add another named export

```js
export const foo = 'bar';
export const baz = 'quux';
```

you must
- remove the export keyword next to your original declaration
- add an export keyword to the bottom of your file
- copy the names of the named exports to the new grouped export

because the end result must be:

```js
const foo = 'bar';
const baz = 'quux';

export { foo, baz };
```

So I want to just have the exporting grouped in one spot regardless of if there's one named export or multiple. What else might I be missing from the explanation here?

**Comment by shoelessness** [comment]
It would help a lot, for sure, but I still kind of see it as a bit of debt to pay off. If you're used to the pattern globally, you just scan the bottom of the file for the exports.

Should I open a separate issue for autofixing the `group-exports` rule?

**Comment by ljharb** [comment]
I don't think an issue is needed; I'd say that two PRs would be welcome here:
1. make `group-exports` fully autofixable, if possible
1. add an option to `group-exports` that allows grouped individual export declarations even for "more than one export"

**Comment by ramirezjag00** [comment]
If anyone is still looking for a fix to just group exports at the bottom of the file, I created a different approach with bash. It's not the greatest but I guess it works for some cases. Please check [this](https://github.com/ramirezjag00/export-sanitize-script) out.

**Comment by ljharb** [comment]
Since that isn't using a proper JS parser and thus will likely fail on a ton of use cases, I'm going to hide it. A PR that adds this behavior to this plugin would be more than welcome.
